### PR TITLE
Updates page titles

### PR DESCRIPTION
**Pull request by DanielCouzens**
Please see ticket https://github.com/epimorphics/ppd-explorer/issues/220

Updates page titles to be more descriptive

### CODE REVIEW

File: app/views/ppd/index.html.haml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 -# frozen_string_literal: true
 
+= content_for(:title, "Search the price paid dataset".html_safe)
```
Comment (jonrandahl) [inline app/views/ppd/index.html.haml]: this also doesn't need `.html_safe` as there's nothing to make safe

File: app/views/search/_search_results.html.haml [hunk 2]
```diff
@@ -1,4 +1,5 @@
 -# frozen_string_literal: true
+= content_for(:title, "Search results".html_safe)
```
Comment (jonrandahl) [inline app/views/search/_search_results.html.haml]: ditto on `.html_safe` here too

**Review by jonrandahl — CHANGES_REQUESTED** [review]
Small minor changes here too! 😃

File: app/views/ppd/index.html.haml (hunk 1 above)
Comment (DanielCouzens) [reply app/views/ppd/index.html.haml]: :+1:

File: app/views/search/_search_results.html.haml (hunk 2 above)
Comment (DanielCouzens) [reply app/views/search/_search_results.html.haml]: :+1:

**Review by jonrandahl — APPROVED** [review]
LGTM! 🚀

**Pull request merged**